dataverse-webapp/src/main/webapp/search/advanced.xhtml

```diff
@@ -242,7 +242,21 @@
                                     </label>
                                     <div class="col-sm-6">
                                         <p:inputText id="fileFieldName" styleClass="form-control"
-                                                     value="#{fileSearchFields.fieldValue}"/>
+                                                     value="#{fileSearchFields.fieldValue}"
+                                                     rendered="#{fileSearchFields.searchFieldType == fieldTypeEnums.TEXT}"/>
+                                        <p:selectCheckboxMenu value="#{fileSearchFields.checkedFieldValues}"
+                                                              rendered="#{fileSearchFields.searchFieldType == fieldTypeEnums.CHECKBOX}"
+                                                              style="width: 100%" panelStyleClass="without-select-all"
```

Comment:
Better to use css class (`styleClass="…"`) here instead of `style`, `form-control` sets the width to 100%.